core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -41,29 +42,36 @@ export class ResultPanelComponent implements OnInit {
   currentOperatorId?: string | undefined;
 
   showResultPanel: boolean = false;
-  previewWorkflowVersion: boolean = false;
+  previewWorkflow: boolean = false;
 
   constructor(
     private executeWorkflowService: ExecuteWorkflowService,
     private resultPanelToggleService: ResultPanelToggleService,
     private workflowActionService: WorkflowActionService,
     private workflowResultService: WorkflowResultService,
     private workflowVersionService: WorkflowVersionService,
-    private changeDetectorRef: ChangeDetectorRef
+    private changeDetectorRef: ChangeDetectorRef,
+    private workflowExecutionService: WorkflowExecutionsService
   ) {}
 
   ngOnInit(): void {
     this.registerAutoRerenderResultPanel();
     this.registerAutoOpenResultPanel();
-    this.handleResultPanelForVersionPreview();
+    this.handleResultPanelForPreview();
```

Comment:
unlike version preview, we need to show the results in the execution preview. I don't think we need to do any of this check

Reply:
have removed changes in result-panel